src/ucar/unidata/apps/simple/latestmodels.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<catalog xmlns="https://www.unidata.ucar.edu/namespaces/thredds/InvCatalog/v1.0" xmlns:xlink="http://www.w3.org/1999/xlink" name="Latest Unidata IDD Model Data" version="1.0">

```

Comment:
Here again the Unidata Namespace URIs should be changed back to "http:" and the XLink ones should not be changed to "https:".